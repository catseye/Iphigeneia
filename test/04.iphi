(* Test 'if ... then ... else' command with negative result *)

var i in var j in begin
    i := 2;
    if i > 4 then
      j := i * 2
    else
      j := i + 1
end
