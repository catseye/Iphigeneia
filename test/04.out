j=3
i=2
